.github/workflows/build.yml

```diff
@@ -86,8 +86,8 @@ jobs:
             # TODO: Fix these broken tests on Windows
             ctest_args: --exclude-regex '^AutoDJProcessorTest.*$'
             cpack_generator: WIX
-            compiler_cache: clcache
-            compiler_cache_path: ${{ github.workspace }}\clcache
+            compiler_cache: sccache
+            compiler_cache_path: C:\Users\runneradmin\AppData\Local\Mozilla\sccache\cache
```

Comment:
Did you try the environment variable LOCALAPPDATA instead of the hard-coded absolute path?

Reply:
Yes. It did not work.

Reply:
When I tried that, the cache restoration hit the key but the restored cache was empty.